tests/test_identifiers.py

```diff
@@ -21,5 +21,14 @@ def test_experimenter_missing_fields(self):
             Person()
 
 
+class TestAssetPath(unittest.TestCase):
+    """Test AssetPath class"""
+
+    def test_asset_path(self):
+        """Test AssetPath with valid path"""
+        asset_path = AssetPath("/path/to/file.txt")
+        self.assertEqual(str(asset_path), "/path/to/file.txt")
```

Comment:
This is looking good - these work also and might be worth documenting
```suggestion
        asset_path = AssetPath(Path("/path/to/file.txt"))
        self.assertEqual(str(asset_path), "/path/to/file.txt")

        asset_path = AssetPath(PureWindowsPath(r"path\to\file.txt"))
        self.assertEqual(str(asset_path), "path/to/file.txt")
```